content/en/dashboards/widgets/log_stream.md

```diff
@@ -5,42 +5,6 @@ widget_type: log_stream
 description: "Display a filtered log stream in your Datadog dashboards."
 aliases:
 - /graphing/widgets/log_stream/
-further_reading:
-- link: "/logs/explorer/analytics/"
-  tag: "Documentation"
-  text: "Log Analytics"
-- link: "/dashboards/graphing_json/"
-  tag: "Documentation"
-  text: "Building Dashboards using JSON"
-
 ---
 
-The Log Stream displays a log flow matching the defined query.
-
-## Setup
-
-{{< img src="/dashboards/widgets/log_stream/log_stream_config.png" alt="Log stream configuration filtered by source:nodejs with three columns for date, host, and service " style="width:100%;" >}}
-
-### Configuration
-
-1. Enter a [search query][1] to filter the log stream.
-1. You can group logs into event subsets of [Log Analytics][2], which include `Patterns` and `Transactions`.
-1. Update the columns to display the values of your [facets][3] and [measures][4].
-1. Give your graph a title or leave the box blank for the suggested title.
-
-## API
-
-This widget can be used with the **[Dashboards API][5]**. See the following table for the [widget JSON schema definition][6]:
-
-{{< dashboards-widgets-api >}}
-
-## Further Reading
-
-{{< partial name="whats-next/whats-next.html" >}}
-
-[1]: /logs/explorer/search/
-[2]: /logs/explorer/analytics/
-[3]: /logs/explorer/facets/
-[4]: /logs/explorer/facets/#measures
-[5]: /api/latest/dashboards/
-[6]: /dashboards/graphing_json/widget_json/
+<div class="alert alert-warning">The Log Stream widget is supported through the <a href="https://docs.datadoghq.com/dashboards/widgets/list/">List widget</a>.</div>
```

Comment:
```suggestion
<div class="alert alert-warning">View the Log Management stream through the <a href="https://docs.datadoghq.com/dashboards/widgets/list/">List widget</a>.</div>
```
What's the purpose of keeping this page around? Why not delete it entirely, and add a note to the List widget page saying you can use it to view the log stream?

Reply:
I'm not sure, this is how the Event stream and event timeline widgets were handled when they got folded into the list widget. But I could add an alias and redirect all three pages to the List page and delete, but I'll confirm with the product team.